CWE/decisions/italy.txt

```diff
@@ -0,0 +1,95 @@
+political_decisions = {
+
+	ina_casa = {
+		picture = "piano_fanfani"
+		potential = {
+		    year = 1949
+			tag = ITA
+			government = democracy
+			has_country_flag = ITA_1948_republic_flag
+			NOT = { has_country_flag = italy_dirigisme }
+			NOT = { has_country_flag = ITA_Piano_Fanfani }
+		}
+		allow = {
+			OR = { ruling_party_ideology = conservative ruling_party_ideology = big_tent }
+			money = 200000
+			is_vassal = no
+		}
+		effect = {
+			prestige = 10
+			money = -100000
+			political_reform = social_abuse_protection
```

Comment:
Please put a check for social abuse protection or better to not have already been enacted.